committees/account_pages/views.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.views.generic import TemplateView
+
+from membership_file.util import MembershipRequiredMixin
+
+from user_interaction.account_pages.mixins import AccountTabsMixin
+
+
+class AssociationGroupAccountView(MembershipRequiredMixin, AccountTabsMixin, PermissionRequiredMixin, TemplateView):
```

Comment:
```suggestion
class AssociationGroupAccountView(AccountTabsMixin, PermissionRequiredMixin, TemplateView):
```
The template used here contains a specific message if the user is not a member, so non-members can access this page too at the moment.